app/liquid_tags/forem_tag.rb

```diff
@@ -0,0 +1,52 @@
+module ForemTag
+  REGISTRY_REGEXP = %r{#{URL.url}/\w+(/[\w-]+)?(/[\w-]+)?}
```

Comment:
I suspect that we really need only check if text matches the URL.url.  All other "slugs" will get picked up by the `determine_klass` method.